src/presentation/components/team/Member.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <div class="team">
+    <div class="team-header">
+      <h1 class="team-title">
+        {{ teamTitle }}
+      </h1>
+    </div>
+    <p class="team-sub">
+      {{ teamSubTitle }}
+    </p>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { TeamMember } from '@/domain/entities/Team.ts';
+import { computed } from 'vue';
+
+/**
+ * NoteListItem props, receive Note
+ */
+const props = defineProps<{
+  teamMember: TeamMember;
+}>();
+
+/**
+ * Get the title like userId
+ */
+const teamTitle = computed(() => {
+  const firstNoteBlock = props.teamMember.noteId;
+
+  if (!firstNoteBlock) {
+    return 'Loading...';
+  } else {
+    return firstNoteBlock;
+  }
+});
+
+/**
+ * Get the subtitle depending on the type of block from Note
+ */
+const teamSubTitle = computed(() => {
+  const secondNoteBlock = props.teamMember.userId;
+
+  if (!secondNoteBlock) {
+    return 'Loading...';
+  } else {
+    return 'User id is ' + secondNoteBlock;
+  }
+});
+</script>
+
+<style>
+.note {
+  margin: 20px 0;
+  padding-bottom: 20px;
+  border-bottom: 1px solid #4b4b4b;
+  cursor: pointer;
+}
+
+.note-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.note-title {
+  font-size: 24px;
```

Comment:
We have vars for this

Reply:
use [typography](https://github.com/codex-team/notes.web/blob/main/codex-ui/src/styles/typography.pcss)